advanced_functionality/autogluon-tabular-containers/AutoGluon_Tabular_SageMaker_Containers.ipynb

```diff
@@ -0,0 +1,735 @@
+{
```

Comment:
<p>Any cleanup needed? </p><p> </p><p>Maybe add a summary section - what was accomplished? What should the user take away from this notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2957/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
No cleanup is needed for batch transform: the job will create an instance, run transformation, then shut down the instance. Added conclusion and next steps information.